Dexterous-Rambutan/battle-code
# Feature admin priv

#### What's this PR do?

Requires admin privilege to access routes: /api/resetDB, /api/resetDBWithData, and /addProblemsSolutions.html
#### What are the important parts of the code?

users/adminPrivilege.js
#### How should this be tested by the reviewer?
1. Open 127.0.0.1:3000
2. Make sure you're not logged in
3. Try to navigate to 127.0.0.1:3000/addProblemsSolutions.html
4. Should see an error message denying access
5. Navigate to 127.0.0.1:3000, then log in
6. Got step 3, but this time, should see the 4 boxes
#### Is any other information necessary to understand this?

Added a middleware called adminPrivilege, that's required for those 3 routes
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

Closes #73 
#### Screenshots (if appropriate)
